release/config.ini.sample

```diff
@@ -252,6 +252,21 @@ packetThreads=2
 # and includes subdomains (the app must be served over https)
 #hstsHeader=true
 
+# Business Hours
+# If set, displays a colored background on the sessions timeline graph during business hours
+# Business hours are assumed to be on a daily basis Monday through Friday
+# Values are set in hours from midnight (default is off)
+#businessDayStart=9
+#businessDayEnd=17
+
+# Business Days
```

Comment:
@awick make sense?

Reply:
Assume this is UTC, should mention that.  It would be could be a range in 1 variable instead of 2. not sure if we are using moment or something else that could do magic.

Reply:
Yes! It is. I'll add a note about that